fflas-ffpack/ffpack/ffpack_rankprofiles.inl

```diff
@@ -30,8 +30,16 @@ namespace FFPACK{
     template <class Field>
```

Comment:
You should apply the same edits to the `ReducedRowRankProfile` and `ReducedColumnRankProfile` routines

Reply:
I did not find any of the 2 routines, perhaps you meant either ColumnRankProfile or RowRankProfile, isn't it ?

Reply:
Of course, I got confused between rank profiles and echelon forms!
Forget about this request!